src/main/common/utils/numericUtils.ts

```diff
@@ -14,3 +14,37 @@ export function toNumberOrUndefined (value: any): number {
 
   return isNaN(numberVal) ? undefined : numberVal
 }
+
+export function containsAValidComma (input: string): boolean {
```

Comment:
Please do not export function. It does not look like something reusable outside.

Reply:
Also please please refer to this as a "thousands separator". "valid comma" sounds a little confusing and does not carry much information.